tools/privacyidea-schema-upgrade

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+
+MIGRATION_PATH=$1
+FIRST_VERSION="4f32a4e1bf33"
+
+if [ -z "$MIGRATION_PATH" ]; then
+	echo "Please specify the path with the migration files."
+	exit 1
+fi
+
+# check if we do not have DB versioning, yet.
+if [ -z "$(pi-manage db current -d /usr/lib/privacyidea/migrations | grep "(head)$")" ]; then
```

Comment:
Hm, I guess this should probably refer to ``MIGRATION_PATH``?

Reply:
done